benchmarks/src/tpch/run.rs

```diff
@@ -42,8 +42,17 @@ use structopt::StructOpt;
 
 use super::{get_tbl_tpch_table_schema, get_tpch_table_schema, TPCH_TABLES};
 
-/// Run the tpch benchmark
+/// Run the tpch benchmark.
+///
+/// This benchmarks is derived from the [TPC-H][1] version
```

Comment:
I moved the details of what the benchmark was doing into the binary from the README, which I think is better as it is closer to the code, but I don't feel strongly about this and would welcome other opinions on the matter